components/consents/api/src/main/java/org/phenotips/consents/internal/PhenoTipsPatientConsentManager.java

```diff
@@ -163,16 +187,48 @@ private Consent fromXWikiConsentConfiguration(BaseObject xwikiConsent, XWikiDocu
             boolean required = intToBool(xwikiConsent.getIntValue("required"));
             boolean affectsFields = intToBool(xwikiConsent.getIntValue("affectsFields"));
             List<String> formFields = null;
+            List<String> dataFields = null;
             if (affectsFields) {
-                formFields = xwikiConsent.getListValue("fields");
+                dataFields = new LinkedList<>();
+                formFields = xwikiConsent.getListValue(FIELDS);
+                fetchConsentFields(formFields, dataFields, xwikiConsent);
```

Comment:
You forgot to remove ```xwikiConsent``` from the method call :).

Reply:
right, fixed.